chapters/101.rst

```diff
@@ -22,9 +24,14 @@ Document
 	:width: 25%
 
 
-Documents are pieces of data in any dataset you want to search, and the input queries you use to find what you want. Basically, they are the input and output data for your search workflows.
+Documents are pieces of data in any dataset you want to search, and the input queries you use to find what you want. Basically, they are the input and output data for your search workflows, also used in Jina Flow.
```

Comment:
They're not **also** used in Jina Flow (as well as search workflows) because search workflows == Jina Flows.

So we can just leave this paragraph as it was before